### ISSUE TITLE
fine tune results on 100 epochs

### ISSUE THREAD

**Issue by r-isr** [wangsr126/MAE-Lite#6]
Hello,

could you please tell the expected accuracy for 100 epochs fine tuning on ImageNet? (after MAE pretrain for 400 epochs, or moco)

I have seen accuracy results after fine tuning for 300 epochs or 1K.

Thanks

**Comment by wangsr126** [comment]
on IN1k:
MAE-pretrain-400ep+finetune-100ep: 76.2 top1 acc.
